plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/GlueIcebergTableOperations.java

```diff
@@ -128,9 +131,18 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
         try {
             stats.getUpdateTable().call(() -> glueClient.updateTable(updateTableRequest));
         }
-        catch (ConcurrentModificationException e) {
+        catch (ConcurrentModificationException | ResourceNumberLimitExceededException e) {
```

Comment:
Is there any point to retry ResourceNumberLimitExceededException? Isn’t that a permanent error (at least until something external cleans it up)?

Reply:
Looks like the only relevant quota is number of versions per table, which would need to be cleaned up manually: https://docs.aws.amazon.com/general/latest/gr/glue.html

Reply:
Correct. 

This probably needs manual intervention from an administrator.

Thank you both for the explanation.